src/main/resources/api/service_openapi.yaml

```diff
@@ -1,264 +1,231 @@
-openapi: 3.0.0
+openapi: 3.0.3
 info:
   title: Workspace Manager API
-  description: |
-    Workspace Manager REST API to manage workspaces.
+  description: Workspace Manager REST API to manage workspaces.
   version: 0.0.1
 
-servers:
-- url: /
-
 paths:
-  '/status':
+  # Unauthorized Paths
+  /status:
     get:
-      description: |
-        Returns the operational status of the service
+      summary: Returns the operational status of the service
       operationId: serviceStatus
-      tags:
-      - unauthenticated
+      tags: [Unauthenticated]
```

Comment:
These are equivalent list formats, so I'm using brackets whenever it's just a list of primitives. The capitalization doesn't change anything in the generated code because it's automatically camel-cased, so it only affects the swagger ui page.